New "Dedicated resource" page (#4285)
umbraco/UmbracoDocs: Umbraco-Cloud/Set-Up/Dedicated/index.md

---
versionFrom: 8.0.0
versionTo: 10.0.0
---

# Move your Umbraco Cloud project to dedicated resources

> 

Reviewer: How about we add this sentence here at the top, in order to make it clear what the terms mean, and give an overview of the feature?

```suggestion

*Dedicated Resources* is a feature on Umbraco Cloud that gives you the option to move your project to a dedicated server. You can choose between a number of *dedicated plans* depending on the amount of resources you will need for your project.

```

## How to move from shared to dedicated

The first step in moving to a dedicated resource is to access your project in the project overview at [Umbraco.io](https://www.s1.umbraco.io/projects).

- In the project overview you can find all the projects that you have been invited to or have created.

- From here you need to find the project that you want to move to dedicated.

- Select *Dedicated Resources* from the *Settings* menu.

![Upgrade plan step 1](images/Step1.png)

- For a Umbraco Cloud project with a Shared Professional plan there are three dedicated options for you to choose among. For each of the dedicated options, you will find its name, the memory and CPU cores, and the price per month.

![Upgrade plan step 1](images/Step2.png)

- By hitting the "Upgrade" button on your dedicated option of choice and confirming this, you will be redirected to the project page where you will be notified when the move to a dedicated resource has been completed.

> :::note Dedicated Resources are reserved on a per-month basis.
> The price of the dedicated resource will take effect from the next period of your subscription.
> If you move your cloud project to a dedicated resource in the middle of the month, the time from that date until the start of the next subscription period, will be added to the next invoice. :::

Reviewer: ```suggestion
:::note 
Dedicated resources are reserved on a per-month basis.
The price of the dedicated resource will take effect from the next period of your subscription.
If you move your Cloud project to a dedicated resource in the middle of the month, the time from that date until the start of the next subscription period, will be added to the next invoice. 
:::
```

## How to move from dedicated to shared

> If you’d like to move from dedicated to shared resources, you can easily do this on the _Dedicated resource_ page.

Reviewer: ```suggestion
Moving away from dedicated resources and back to shared can be done from the *Dedicated Resources* page.
```

![Upgrade plan step 1](images/Downgrade.png)

- By hitting "Downgrade to shared" and confirming your choice, you will be redirected to the project page where you will be notified when the move back to a shared resource has been completed.

> - That's it! You cloud project is now back on a shared resource.

Reviewer: ```suggestion
- Your Cloud project is now back on a shared resource.
```

---
versionFrom: 8.0.0
versionTo: 10.0.0
---

# Move your Umbraco Cloud project to dedicated resources

> In this article, you can read about how you enable your Umbraco Cloud project to run on dedicated resources and what you need to be aware of before you do so.

Reviewer: ```suggestion
In this article, you can read about how you can move your Umbraco Cloud project to dedicated resources and what you need to be aware of before you do so.
```

## Before you move your project to dedicated resources

Before you decide to move your Umbraco Cloud project, you need to consider a few things:

> - Currently, Umbraco Cloud offers dedicated resources for professional plans only. We expect to include dedicated resources for Starter and Standard plans in the self-service feature in Umbraco Cloud portal later in 2022.

Reviewer: ```suggestion
- Currently, Umbraco Cloud offers dedicated resources for Professional plans only. We expect to include dedicated resources for Starter and Standard plans in the self-service feature in Umbraco Cloud portal later in 2022.
```

> - In case you want to move your Umbraco Cloud project with a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.

Reviewer: ```suggestion
- In case you want to move a project on a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.
```

> - In case you want to move a project on a Starter or Standard project plan to a dedicated server in Umbraco Cloud, please reach out to our support who will assist you in doing this.

Reviewer 2: I'd change this to:

- Projects on Starter and Standard plans can upgrade to Professional to take advantage of Dedicated Resources, or alternatively, contact support for additional options.

Reviewer 2: Aside from the above, it looks really good 👏

> - Moving from a shared plan to a dedicated plan will change the outgoing IP of the project. If your solution has an external service that requires whitelisting the outgoing IP of the project, we advise you to enable the static outbound IP feature for your cloud project and share that static outbound IP address with the third party. The static outbound IP address will not change when moving from a shared plan to a dedicated resource. For more info on static lease visit the documentation for [external services](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/External-Services/).

Reviewer: ```suggestion
- Moving from a shared plan to a dedicated plan will change the outgoing IP of the project. If your solution has an external service that requires whitelisting the outgoing IP, we advise you to enable the static outbound IP feature for your project and share that static outbound IP address with the third party. The static outbound IP address will not change when moving from a shared plan to a dedicated resource. For more info on static lease visit the documentation for [external services](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/External-Services/).
```

Reviewer: Also, "static lease" - what does that mean exactly?
Is that the static IP? It's important to be consistent with the wording, so is there any chance it would make sense to say something like:
"For more info on static IPs, visit the documentation ... " ?

Author: Hmm, seems like some text unintentionally was deleted in my PR. I believe the wording should be "For more info on static outbound IP please...". How can I change this while this PR is still active? :-)

## How to upgrade your plan

> The first step moving to a dedicated resource is to access your project in the project overview at [Umbraco.io](https://www.s1.umbraco.io/projects).

Reviewer: ```suggestion
The first step in moving to a dedicated resource is to access your project in the project overview at [Umbraco.io](https://www.s1.umbraco.io/projects).
```

> - In the project overview you can find all the projects that you have been invited to or have created.
> 
> - From here you need to find the project that you want to move to dedicated.

Reviewer: ```suggestion
- Find and select the project that you want to move to dedicated resources.
```

> Under the project on the right side you have a dropdown menu called settings:
> 
> ![Upgrade plan step 1](images/Step1.png)
> 
> - Clicking on the tab will direct you to the _Dedicated resource_ page that guides you on how to upgrade to a dedicated plan.

Reviewer: ```suggestion
- Select *Dedicated Resources* from the *Settings* menu.

![Upgrade plan step 1](images/Step1.png)
```

> - For a Umbraco Cloud project with a Shared Professional plan there are three dedicated plans for you to choose among. For each of the dedicated plans, you will find its name, the memory and CPU cores, and the price per month.

Reviewer: Does this mean that there are other options for other plans? 🤔 
As the feature is only available for pro plans at the moment, I think we can leave out the part about how this particular plan has these specific options.

We can always specify the options when we expand the feature.

Author: I am not sure that I entirely understand the reasoning for not having this info that there are three dedicated plans to choose from.

Author: In the latest commit, I changed the term "dedicated plan" to "dedicated option" (and "shared plan" to "shared resource") as this is a better term and won't collide with our normal use of the "plan". That will help quite a bit with the understanding, I believe :-).

We definitely could add a "currently" in there as well, if that will help.
